website/content/api-docs/system/internal-counters.mdx

```diff
@@ -89,9 +89,11 @@ $ curl \
 
 ## Client Count
 
-This endpoint returns the number of clients per namespace, as the sum of active entities and non-entity tokens.
+This endpoint returns the number of clients per namespace, as the sum of active entities and non-entity clients.
 An "active entity" is a distinct entity that has created one or more tokens in the given time period.
-A "non-entity token" is a token with no attached entity ID.
+A "non-entity client" is one that authenticates to vault using a "non-entity token", i.e. a token with
+no attached entity ID. For more information on how non-entity clients are counted, please visit the 
+[client count](https://www.vaultproject.io/docs/concepts/client-count) concepts page.
```

Comment:
The link continues to use the terminology "non-entity token".  Do we have consensus for the new "non-entity client" terminology you're introducing?

Reply:
Regarding this, I've sync'ed with the PM team on terminology. For 1.9, we'll keep the terminology `non-entity token` and explain that the client counting for the `non-entity token` section of the API results have essentially been updated. I'll update the docs to reflect that!

Reply:
Do we want to add a line here that mentions that while there is no attached entity ID to non-entity tokens, we now have a clientID attached to them?

Reply:
Do we want to add a line here that mentions that while there is no attached entity ID to non-entity tokens, we now have a clientID attached to them?

Reply:
Done! Does that look ok?

Reply:
not sure if you care, but you can also link the relative path instead of the web (e.g. `/docs/concepts/client-count`)